app/models/patron.rb

```diff
@@ -94,7 +94,11 @@ def fines
   end
 
   def requests
-    @requests ||= fields['holdRecordList'].map { |request| Request.new(request) }
+    @requests ||= if fields['holdRecordList'].nil?
```

Comment:
I think Rails also gives us the ability to use `Array.wrap` which could remove our `nil?` check.

@requests ||= Array.wrap(fields['holdRecordList']).map { |request| Request.new(request) }
```suggestion
    @requests ||= Array.wrap(fields['holdRecordList']).map { |request| Request.new(request) }
```